test/integration/tests/pcrextend.sh

```diff
@@ -14,8 +14,7 @@ declare -A alg_hashes=(
 
 digests=""
 # test a single algorithm based on what is supported
-for alg in `tpm2_getcap pcrs | sed -r -e '1d' -e s/'\s+-\s+(\w+):.*'/'\1'/g`; do
-  alg=`echo $alg | cut -d\( -f 1-1`;
+for alg in `tpm2_getcap pcrs | awk {'print $2'} | cut -d: -f 1`; do
```

Comment:
@tstruk for purpose of portability, should the cut be an awk instead?

Reply:
@idesai updated to use awk